File: Anul 2/Semestrul 2/MPP/labs/Tema 5-proiect Server Csharp-Client Java/proiectSCcsharp/angajatModel/Bilet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace proiectSCcsharp.domain
{
    [Serializable]

    public class Bilet : Entity<long>
    {
        public string cumparatorName { get; set; } 
        public long idSpectacol { get; set; }  
        public int nrSeats { get; set; }

        public Bilet(long id, string CumparatorName, long Id_spectacol, int NrSeats):base(id)
        {
            cumparatorName = CumparatorName;
            idSpectacol = Id_spectacol;
            nrSeats = NrSeats;
        }
        public override string ToString()
        {
            return cumparatorName + " | " + idSpectacol + " | " + nrSeats ;
        }
    }
}

File: Anul 2/Semestrul 2/MPP/labs/Tema 7-servicii REST/Java/Rest/proiect/Domain/src/main/java/proiect/domain/Bilet.java
package proiect.domain;

import java.io.Serializable;

public class Bilet extends Entity<Integer> implements Serializable {
    private String cumparatorName;
    private int idSpectacol;
    private int nrSeats;

    public Bilet(Integer id, String name, int id_spectacol, int nrSeats)
    {
        super(id);
        this.cumparatorName=name;
        this.idSpectacol=id_spectacol;
        this.nrSeats=nrSeats;
    }

    public String getCumparatorName() {
        return cumparatorName;
    }

    public void setCumparatorName(String cumparatorName) {
        this.cumparatorName = cumparatorName;
    }

    public int getIdSpectacol() {
        return idSpectacol;
    }

    public void setIdSpectacol(int id_spectacol) {
        this.idSpectacol = id_spectacol;
    }

    public int getNrSeats() {
        return nrSeats;
    }

    public void setNrSeats(int nrSeats) {
        this.nrSeats = nrSeats;
    }
}
